mailbox/opensearch/src/main/java/org/apache/james/mailbox/opensearch/query/DefaultCriterionConverter.java

```diff
@@ -85,14 +85,14 @@ public DefaultCriterionConverter() {
         registerHeaderOperatorConverters();
     }
 
-    private String evaluateFuzzinessValue(boolean textFuzzinessSearchEnable) {
+    protected String evaluateFuzzinessValue(boolean textFuzzinessSearchEnable) {
```

Comment:
Does that really make sense in the end? Would not implement the interface anymore but extend DefaultCriterionConverter instead then?

Reply:
Limiting the code duplication is IMO desirable.

extend DefaultCriterionConverter in tmail is BTW not incompatible with interface in tmail

BTW it would ease injects...